### ISSUE TITLE
Documentation: How does translation work

### ISSUE THREAD

**Issue by tordans** [openstreetmap/id-tagging-schema#55]
I was looking at https://github.com/openstreetmap/id-tagging-schema to find the link about where to translate things. Its probably at https://www.transifex.com/openstreetmap/id-editor/translate/#de/presets/30538703?q=text%3A'Hearing+Aids+Store'. A section with the link and about how it works in the readme would be great, IMO.

**Comment by tordans** [comment]
Another detail for the documentation:

Should terms for english be put in Transifex as part of the .terms translation (eg https://www.transifex.com/openstreetmap/id-editor/translate/#en_GB/presets/30538703?q=text%3A'Aids+Store') or should they be put in the JSON like https://github.com/openstreetmap/id-tagging-schema/blob/bf95421f65018ccd7f398f8459b37462febddcba/data/presets/natural/shrub.json#L16-L19?

**Comment by quincylvania** [comment]
@tordans Yep, the translations continue to live at the id-editor Transifex project for now. I'll update the readme.

> Should terms for english be put in Transifex as part of the .terms translation (eg https://www.transifex.com/openstreetmap/id-editor/translate/#en_GB/presets/30538703?q=text%3A'Aids+Store') or should they be put in the JSON like 

The US English terms go in the JSON data. If we need different terms for a different English locale, those can be translated on Transifex. But it's usually better to put local terms from the UK or other speaking countries directly in the US terms.

**Comment by peternewman** [comment]
Is #200 having been merged sufficient to close this, or do people want all the detail of #245 to go in first?

I wonder if a lot of the detail from #245 is sufficiently generic enough it could just live at the destination of the link (i.e. in the main iD repo)?

**Comment by tordans** [comment]
@peternewman I have been working on clarifying the process to new users in https://github.com/openstreetmap/id-tagging-schema/pull/245 for a while now and would very much like to see it merged. The questions in the PR are thinks that I wonder myself and that one cannot find the answer to unless asking. I do not see any reason not to finish the PR.
